### PR TITLE
Add attach tip modal to step-1 of deck calibration

### PR DESCRIPTION
**Pull request by Kadee80**
## overview
This PR addresses UI side of #1237 (checkbox 3). Render an attach tip modal prompt after move to attach tip position. 
- Does not include back button functionality
- Does not include onConfirmClick
- Does not include pipette diagram display logic

screen shot 2018-05-01 at 11 19 44 am

## changelog
- Add attach tip modal to step-1 route
- Update Modal Page to style optional heading prop
- Placeholder logic for props to render `ClearDeckAlertModal` and Initial `RequestInProgressModal`

## review requests
This is all placeholder state/render logic for slugging in UI components. `ClearDeckAlert` renders by default, to see `AttachTipModal` go to `app/src/components/CalibrateDeck/index.js` and comment out lines 78-90.

### CODE REVIEW

File: app/src/components/CalibrateDeck/styles.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.attach_tip_instructions {
+  @apply --font-body-2-dark;
+
+  max-width: 60%;
+
+  & > p {
+    margin: 1rem 0;
+  }
+
+  & > button {
```
Comment (mcous) [inline app/src/components/CalibrateDeck/styles.css]: We should keep an eye on this in case the button becomes a link

Comment (Kadee80) [reply app/src/components/CalibrateDeck/styles.css]: agreed. The image display logic will likely change as well once we conditionally display a single channel asset as well.

File: app/src/components/CalibrateDeck/index.js [hunk 2]
```diff
@@ -76,3 +102,10 @@ function mapDispatchToProps (dispatch: Dispatch, ownProps: OP): DP {
     back: () => dispatch(goBack())
   }
 }
+
+function makeMapStateToProps (state: State, ownProps: OP): SP {
```
Comment (mcous) [inline app/src/components/CalibrateDeck/index.js]: Technically this is wrong but I'm following up in another PR so it'll be easier if we just merge. Should be:

```js
function makeMapStateToProps () {
  const getRobotMove = makeGetRobotMove()

  return (state: State, ownProps: OP): SP => ({
    moveRequest: getRobotMove(state, ownProps.robot)
  })
}
```

**Comment by Kadee80** [comment]
Addresses #1320

**Pull request merged**